.github/workflows/iroha2-dev-pr-label.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches: [iroha-dev]
     paths:
       - 'docs/source/references/schema.json'
+      - 'configs/**'
```

Comment:
![image](https://github.com/hyperledger/iroha/assets/49983831/b6494fd0-12a0-47cd-95bd-bc99e08af251)
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags